### ISSUE TITLE
NTR: Phosphoethanolamine high in blood (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8519]
New term request Phosphoethanolamine high in blood (segal_030505173132): Ethanolamine phosphate; O-Phosphorylethanolamine; O-Phosphoethanolamine

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide additional context? Does this differ from 
[Elevated circulating o-phosphoethanolamine concentration HP:0033406](https://hpo.jax.org/app/browse/term/HP:0033406)

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/146300 "Serum PLP and urine phosphoethanolamine (PEA) were abnormally high in both of the twins and a definitive diagnosis of hypophosphatasia was made, which was supported by findings in other members of the kindred."

**Comment by pnrobinson** [comment]
AFAICS the two concepts refer to the same thing, the "O" prefix refers to the place where the phosphogroup is attached and does not refer to a second compound.
Here is a review:
https://pubmed.ncbi.nlm.nih.gov/35878747/
I am going to change the primary label of the current term to not have the "o-" and and it as a synonym.
